cumulusci/cli/service.py

```diff
@@ -229,8 +229,13 @@ def service_info(runtime, service_type, service_name, plain):
             ]
         )
         wrap_cols = ["Value"] if not plain else None
+        service_name = (
+            runtime.keychain._default_services[service_type]
```

Comment:
We should avoid directly accessing a structure that is meant to be internal to the keychain implementation. Should we add a method like get_default_service_name to BaseProjectKeychain and move these lines to it? https://github.com/SFDO-Tooling/CumulusCI/blob/main/cumulusci/core/keychain/base_project_keychain.py#L212-L216

That will also make sure this gives a meaningful error if there isn't a default set for some reason.

Reply:
I should have realized this was a code smell when I wrote it. I like the idea of including another Public API method (get_default_service) that can be utilized. I'm doing some similar re-work in the merge-keychain PR so that as few methods as possible are directly mutating the `keychain.config` attribute.